### PR TITLE
Primera vista "Mejor Reseña"

### PR DESCRIPTION
**Pull request by N0isome**
<!DOCTYPE html>
<html lang="en">
<head>
    <title>Mejor Reseña - Golden Grow</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700,800,900" rel="stylesheet">
    
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
    <link rel="stylesheet" href="css/style.css">
    <style>
        /* Agregar estilos adicionales aquí si es necesario */
        .carousel-item img {
            width: 100%;
            height: auto;
        }
    </style>
</head>
<body>
    
    <div class="wrapper d-flex align-items-stretch">
        
        <nav id="sidebar">
            <div class="custom-menu">
                <button type="button" id="sidebarCollapse" class="btn btn-primary">
                </button>
            </div>
            <div class="img bg-wrap text-center py-4" style="background-image: url(images/bg_1.jpg);">
                <div class="user-logo">
                    <div class="img" style="background-image: url(images/logo.jpg);"></div>
                    <h3>Golden Grow</h3>
                </div>
            </div>
            <ul class="list-unstyled components mb-5">
                <li>
                    <a href="index.html"><span class="fa fa-home mr-3"></span> Inicio</a>
                </li>
                <li>
                    <a href="autos.html"><span class="fa fa-car mr-3"></span> Autos</a>
                </li>
                <li class="active">
                    <a href="#"><span class="fa fa-trophy mr-3"></span> Mejor Reseña</a>
                </li>
                <li>
                    <a href="#"><span class="fa fa-cog mr-3"></span> Configuración</a>
                </li>
                <li>
                    <a href="#"><span class="fa fa-support mr-3"></span> Soporte</a>
                </li>
                <li>
                    <a href="#"><span class="fa fa-sign-out mr-3"></span> Cerrar Sesión</a>
                </li>
            </ul>

        </nav>

        
        <div id="content" class="p-4 p-md-5 pt-5">
            <h2 class="mb-4">Mejor Reseña</h2>
            <div class="row">
                <div class="col-md-6">
                    <div class="card">
                        <div class="card-body">
                            <h5 class="card-title">Autos más vendidos de la temporada</h5>
                            <p class="card-text">Según nuestras estadísticas, estos son los autos que más eligen nuestros clientes.</p>
                        </div>
                    </div>
                </div>
                <div class="col-md-6">
                    <div class="card">
                        <div class="card-body">
                            <canvas id="myChart" class="chartjs-render-monitor"></canvas>
                        </div>
                    </div>
                </div>
            </div>
            <h2 class="mt-5">Oferta de la Semana</h2>
            <div id="carouselExampleControls" class="carousel slide" data-bs-ride="carousel" data-bs-interval="3000">
                <div class="carousel-inner">
                    <div class="carousel-item active">
                        Auto 1
                    </div>
                    <div class="carousel-item">
                        Auto 2
                    </div>
                    <div class="carousel-item">
                        Auto 3
                    </div>
                    <div class="carousel-item">
                        Auto 4
                    </div>
                    <div class="carousel-item">
                        Auto 5
                    </div>
                </div>
                <button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleControls" data-bs-slide="prev">
                    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
                    <span class="visually-hidden">Anterior</span>
                </button>
                <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleControls" data-bs-slide="next">
                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
                    <span class="visually-hidden">Siguiente</span>
                </button>
            </div>
        </div>
    </div>

    
    <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
    <script>
        document.addEventListener('DOMContentLoaded', function() {
            var ctx = document.getElementById('myChart').getContext('2d');
            var myChart = new Chart(ctx, {
                type: 'line',
                data: {
                    labels: ['Semana 1', 'Semana 2', 'Semana 3', 'Semana 4', 'Semana 5', 'Semana 6', 'Semana 7', 'Semana 8'],
                    datasets: [{
                        label: 'Autos más vendidos del mes',
                        data: [12, 19, 3, 5, 15, 18, 22, 27],
                        backgroundColor: [
                            'rgba(255, 99, 132, 0.2)',
                        ],
                        borderColor: [
                            'rgba(255, 99, 132, 1)',
                        ],
                        borderWidth: 1
                    }]
                },
                options: {
                    scales: {
                        y: {
                            beginAtZero: true
                        }
                    }

                }
            });
        });
    </script>
</body>
</html>

### CODE REVIEW

**Comment by N0isome** [comment]
Modificacion de la pagina "mejor reseña ":
<!DOCTYPE html>
<html lang="en">
<head>
    <title>Mejor Reseña</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700,800,900" rel="stylesheet">
    
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/apexcharts"></script>
</head>
<body>
    
    <div class="wrapper d-flex align-items-stretch">
        <nav id="sidebar">
            <div class="custom-menu">
                <button type="button" id="sidebarCollapse" class="btn btn-primary">
                </button>
            </div>
            <div class="img bg-wrap text-center py-4" style="background-image: url(images/bg_1.jpg);">
                <div class="user-logo">
                    <div class="img" style="background-image: url(images/logo.jpg);"></div>
                    <h3>Golden Grow</h3>
                </div>
            </div>
            <ul class="list-unstyled components mb-5">
                <li>
                    <a href="index.html"><span class="fa fa-home mr-3"></span> Inicio</a>
                </li>
                <li>
                    <a href="autos.html"><span class="fa fa-car mr-3"></span> Autos</a>
                </li>
                <li class="Mejor Reseña">
                    <a href="#"><span class="fa fa-trophy mr-3"></span> Mejor Reseña</a>
                </li>
                <li>
                    <a href="#"><span class="fa fa-cog mr-3"></span> Configuración</a>
                </li>
                <li>
                    <a href="#"><span class="fa fa-support mr-3"></span> Soporte</a>
                </li>
                <li>
                    <a href="#"><span class="fa fa-sign-out mr-3"></span> Cerrar Sesión</a>
                </li>
            </ul>

        </nav>

        
        <div id="content" class="p-4 p-md-5 pt-5">
            <h2 class="mb-4">Mejor Reseña</h2>
            <div class="row">
                <div class="col-md-6">
                    
                    <div id="carouselExample" class="carousel slide" data-ride="carousel">
                        <div class="carousel-inner">
                            <div class="carousel-item active">
                                Toyota Yaris 2016
                            </div>
                            <div class="carousel-item">
                                Kia Morning 2018
                            </div>
                            <div class="carousel-item">
                                Chevrolet Sail
                            </div>
                            <div class="carousel-item">
                                Toyota Hilux
                            </div>
                            <div class="carousel-item">
                                Hyundai Accent
                            </div>
                        </div>
                        <a class="carousel-control-prev" href="#carouselExample" role="button" data-slide="prev">
                            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
                            <span class="sr-only">Previous</span>
                        </a>
                        <a class="carousel-control-next" href="#carouselExample" role="button" data-slide="next">
                            <span class="carousel-control-next-icon" aria-hidden="true"></span>
                            <span class="sr-only">Next</span>
                        </a>
                    </div>
                </div>
                <div class="col-md-6">
                    <h2>Incremento de Ventas</h2>
                    <p>El incremento de ventas para todos los modelos es del 60% en los últimos 6 meses.</p>
                    <div id="chart"></div>
                </div>
            </div>
        </div>
    </div>

    <script>
        var options = {
            series: [{
                name: 'Incremento de Ventas',
                data: [80, 30, 15, 73, 10] // Incremento ficticio de ventas para cada modelo
            }],
            chart: {
                type: 'bar',
                height: 250
            },
            plotOptions: {
                bar: {
                    horizontal: true,
                }
            },
            dataLabels: {
                enabled: false
            },
            xaxis: {
                categories: ['Toyota Yaris 2016', 'Kia Morning 2018', 'Chevrolet Sail', 'Toyota Hilux', 'Hyundai Accent'],
            }
        };

        var chart = new ApexCharts(document.querySelector("#chart"), options);
        chart.render();
    </script>
</body>
</html>
